services/masterdata/src/lib/components/masterdata-type-edit-form/masterdata-type-edit-form.component.ts

```diff
@@ -68,6 +68,11 @@ export class MasterdataTypeEditFormComponent implements OnInit {
 	loadData() {
 		this.isLoading = true;
 		this.service.getMasterdataTypeById(this.id).subscribe(data => {
+      if (!data) {
```

Comment:
Mixed tabs and spaces. Are your vs code settings correct? Statusbar (on the right) should say TabSize:4

Reply:
Also check the other files

Reply:
i will fix this formatting in a future update, because i used the 2 spaces identation as default!